_posts/2022-03-25-does-adam.md

```diff
@@ -0,0 +1,312 @@
+- [7] P. Isola, J.-Y. Zhu, T. Zhou, and A. A. Efros.  Image-to-image translation with conditionaladversarial networks. InProceedings of the IEEE conference on computer vision and patternrecognition, pages 1125–1134, 2017.
+- [8] D.  P.  Kingma  and  J.  Ba.   Adam:  A  method  for  stochastic  optimization.arXiv  preprintarXiv:1412.6980, 2014.
+- [9] T. P. Lillicrap, J. J. Hunt, A. Pritzel, N. Heess, T. Erez, Y. Tassa, D. Silver, and D. Wierstra.Continuous control with deep reinforcement learning.arXiv preprint arXiv:1509.02971, 2015.
+- [10] L. Luo, Y. Xiong, Y. Liu, and X. Sun.  Adaptive gradient methods with dynamic bound oflearning rate.arXiv preprint arXiv:1902.09843, 2019.- [11]H. B. McMahan and M. Streeter. Adaptive bound optimization for online convex optimization.arXiv preprint arXiv:1002.4908, 2010.
+- [12] V. Mnih, A. P. Badia, M. Mirza, A. Graves, T. Lillicrap, T. Harley, D. Silver, and K. Kavukcuoglu.Asynchronous methods for deep reinforcement learning. InInternational conference on machinelearning, pages 1928–1937. PMLR, 2016.
+- [13] A. Radford, L. Metz, and S. Chintala. Unsupervised representation learning with deep convolu-tional generative adversarial networks.arXiv preprint arXiv:1511.06434, 2015.
+- [14] S. J. Reddi, S. Kale, and S. Kumar. On the convergence of adam and beyond.arXiv preprintarXiv:1904.09237, 2019.
+- [15] J. Schulman, F. Wolski, P. Dhariwal, A. Radford, and O. Klimov. Proximal policy optimizationalgorithms.arXiv preprint arXiv:1707.06347, 2017.
+- [16] C. Seward, T. Unterthiner, U. Bergmann, N. Jetchev, and S. Hochreiter. First order generativeadversarial networks.  InInternational Conference on Machine Learning, pages 4567–4576.PMLR, 2018.
+- [17] N. Shi, D. Li, M. Hong, and R. Sun.  Rmsprop converges with proper hyper-parameter.  InInternational Conference on Learning Representations, 2020.
+- [18] T. Tieleman and G. Hinton. Divide the gradient by a running average of its recent magnitude.coursera: Neural networks for machine learning.Technical Report, 2017.
+- [19] A. Vaswani,  N. Shazeer,  N. Parmar,  J. Uszkoreit,  L. Jones,  A. N. Gomez,  Ł. Kaiser,  andI. Polosukhin. Attention is all you need. InAdvances in neural information processing systems,pages 5998–6008, 2017.
+- [20] S. Vaswani, F. Bach, and M. Schmidt. Fast and faster convergence of sgd for over-parameterizedmodels and an accelerated perceptron.  InThe 22nd International Conference on ArtificialIntelligence and Statistics, pages 1195–1204. PMLR, 2019.
+- [21] M. D. Zeiler.  Adadelta: an adaptive learning rate method.arXiv preprint arXiv:1212.5701,2012.
+- [22] J.-Y. Zhu,  T. Park,  P. Isola,  and A. A. Efros.   Unpaired image-to-image translation usingcycle-consistent adversarial networks. InProceedings of the IEEE international conference oncomputer vision, pages 2223–2232, 2017.
+
+
+
+[[click here to go to the top]](#introduction-and-background)
+
+
+---
```

Comment:
Could you remove this final page divider?

Reply:
> 

Hi, I have deleted the final page divider, please check.